### ISSUE TITLE
using images created by stylegan as training dataset

### ISSUE THREAD

**Issue by zhangxuan1918** [zhangxuan1918/InverseRendering3DMMCnn#68]
model trained with 300W_LP dataset works relatively well on 300W_LP data. There is no overfitting in training process. However when testing with real life faces downloaded from internet, the results are not good. 

With current loss function, we only need landmark for training. We can use images from stylegan as training data. To get landmarks, we can use pretrained model, e.g https://github.com/1adrianb/face-alignment
